spec/services/work_combined_audio_creator_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+describe "Combined Audio" do
+  # We need to set the friendlier_id explicitly, because it's part of the fingerprint recipe.
+  let!(:work) { FactoryBot.create(:work, title: "top", friendlier_id: '1ciyp1i')}
+
+  let!(:audio_asset_1)  { create(:asset, :inline_promoted_file,
+      position: 1,
+      parent_id: work.id,
+      id: 'ddbd1a8d-c2eb-47b3-85a4-11b4ffb41719',
+      file: File.open((Rails.root + "spec/test_support/audio/ice_cubes.mp3"))
```

Comment:
Would not have expected you had to set the `id` manually here, but you did?

Reply:
Only if I needed to check that the same id resulted in the same fingerprint. Since I'm not checking that anymore per your comment above ... no need to set the `id`!

Reply:
Removed it!